yarn-error.log

```diff
@@ -0,0 +1,8413 @@
+Arguments: 
```

Comment:
The yarn error log should not be added. (you can add it to .gitignore to avoid this in the future)